test/openbci-sdk-test.js

```diff
@@ -248,84 +602,658 @@ describe('openbci-sdk',function() {
             });
         });
     });
-    xdescribe('#simulator', function() {
-        var running = false;
-        beforeEach(function(done) {
-            var ourBoard = new openBCIBoard.OpenBCIBoard();
-
-            ourBoard.simulatorStart().then(function() {
-                console.log('Simulator started');
-                ourBoard.on('sample',function(sample) {
-                    //OpenBCISample.debugPrettyPrint(sample);
-                });
-            }).catch(function(err) {
-                console.log('Error [simulator]: ' + err);
-            });
-            setTimeout(function() {
-                ourBoard.simulatorStop().then(function() {
-                    running = true;
-                    done();
-                },function(err) {
-                    console.log('Error: ' + err);
+
+
+});
+
+describe('#impedanceTesting', function() {
```

Comment:
can the describes below be looped over instead of having separate ones per channel? it would make that a bit cleaner especially since everything is the same except the channel number